src/app/app.module.ts

```diff
@@ -14,7 +15,8 @@ import {FaqService} from "./services/faq.service";
     CoreModule,
     PagesModule
   ],
-  providers: [FacebookService, FaqService],
+  providers: [FacebookService, FaqService, GuestsService],
```

Comment:
Again - GuestService has nothing to do with bulding mode. A separate commit would be appropriate.